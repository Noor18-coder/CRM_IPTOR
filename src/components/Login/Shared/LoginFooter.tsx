import React from 'react';

const LoginFooter = (): JSX.Element => {
  return (
    <p className="login-copyrights">
      Copyright &copy; 2021 - Iptor Sweden AB |{' '}
      <button type="button" className="text-link footer-anchor-button">
        Privacy Policy
      </button>{' '}
      |{' '}
      <button type="button" className="text-link footer-anchor-button">
        UK Legal?
      </button>{' '}
      |{' '}
      <button type="button" className="text-link footer-anchor-button">
        Code
      </button>{' '}
    </p>
  );
};

export default LoginFooter;
