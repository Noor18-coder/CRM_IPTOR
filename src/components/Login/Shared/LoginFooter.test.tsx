import renderer from 'react-test-renderer';
import LoginFooter from './LoginFooter';

describe('[Login Shared] LoginFooter', () => {
  it('should renders correctly', () => {
    const tree = renderer.create(<LoginFooter />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
