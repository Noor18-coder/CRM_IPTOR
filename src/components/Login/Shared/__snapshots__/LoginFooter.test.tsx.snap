// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[Login Shared] LoginFooter should renders correctly 1`] = `
<p
  className="login-copyrights"
>
  Copyright Â© 2021 - Iptor Sweden AB |
   
  <button
    className="text-link footer-anchor-button"
    type="button"
  >
    Privacy Policy
  </button>
   
  |
   
  <button
    className="text-link footer-anchor-button"
    type="button"
  >
    UK Legal?
  </button>
   
  |
   
  <button
    className="text-link footer-anchor-button"
    type="button"
  >
    Code
  </button>
   
</p>
`;
